esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -186,7 +186,7 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
                 if(proc_cfg.get() != NULL)
                 {
                     Owned<IPropertyTree> secMgrs;
-                    VStringBuffer sm("SecurityManagers/SecurityManager[@name='%s']", m_authmethod.str());
+                    VStringBuffer sm("SecurityManagers/SecurityManager[@name='%s']/HtpasswdSecurityManager", m_authmethod.str());
```

Comment:
We shouldn't hardcode plugin names in the platform. We should provide the SecurityManager tree rooted at the SecurityManager element, or we might be kind to the plug-in developers and provide the tree rooted at the first child of the SecurityManager tree where @name = m_authmethod.str()